home_assignment/dal/inmemory_database.py

```diff
@@ -14,23 +15,39 @@ def add(self, message: Message):
     def get_all(self) -> List[Message]:
         return self.messages_list
 
-    def get_by_id(self, user_id) -> str:
+    def get_by_id(self, user_id: int) -> str:
         user_messages_list = self.messages_list
         message = list(filter(lambda m: m.user_id == user_id, user_messages_list))
         message[0].read = True
-        return message[0]
-
-    def get_specific_unread(self, user_id) -> str:
+        user_id: str = message[0].user_id
+        name: str = message[0].name
+        title = message[0].title
+        user_message = message[0].message
+        read = message[0].read
+        created_at = message[0].created_at
+        return {f"id: {user_id}, "
+                               f"name: {name}, "
+                               f"title: {title}, "
+                               f"message: {user_message}, "
+                               f"read: {read}, "
+                               f"created_at: {created_at} "
```

Comment:
Why do you use fstring here? you build a dict

Reply:
Replaced with dict